.github/CODEOWNERS

```diff
@@ -0,0 +1,135 @@
+
+
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+*       @rrsettgast
+
+/.github/ @rrsettgast
+
+/inputFiles/ @CusiniM @cssherman @jhuang2601
+
+/src/coreComponents/LvArray                           @rrsettgast @wrtobin
```

Comment:
maybe this is not necessary since the `LvArray` PR will be the one actually containing the changes?

Reply:
yeah...but sometimes people end up accidentally changing the hash